### PR TITLE
moving to openapi-dependencies

### PR DESCRIPTION
**Pull request by AustinAbro321**

### CODE REVIEW

File: schemaLoader.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func (sl *SchemaLoader) validateMetaschema(documentNode interface{}) error {
 // AddSchemas adds an arbritrary amount of schemas to the schema cache. As this function does not require
 // an explicit URL, every schema should contain an $id, so that it can be referenced by the main schema
 func (sl *SchemaLoader) AddSchemas(loaders ...JSONLoader) error {
-	emptyRef, _ := gojsonreference.NewJsonReference("")
+	emptyRef, _ := jsonreference.New("")
```
Comment (bdw617) [inline schemaLoader.go]:
```suggestion
	emptyRef, err := jsonreference.New("")
	if err != nil {
	        return err
	}
```

**Review by bdw617 — CHANGES_REQUESTED** [review]

**Review by lucasrod16 — CHANGES_REQUESTED** [review]
We should update all references to `xeipuuv` in README.md.

Examples:
 - `go get github.com/xeipuuv/gojsonschema` -> `go get github.com/defenseunicorns/gojsonschema`
 
 - `github.com/xeipuuv/gojsonreference` -> `github.com/go-openapi/gojsonreference`
 
 There are many other references to `xeipuuv` throughout the repository. `glide.yaml`, `LICENSE.APACHE-2.0.txt`, and all of the copyright statements in `*.go` files. I'm not sure whether we want to or should update them or not. @Racer159 thoughts?

**Comment by AustinAbro321** [comment]
Ah good point @lucasrod16. Yeah those should be updated. I'm also not sure about the copyright statements. 

I believe I should make the change from `go get github.com/xeipuuv/gojsonschema` to `go get github.com/gojsonschema/gojsonschema` since we plan on contributing back to https://github.com/gojsonschema/gojsonschema

In that case in our code I think we should use the go mod replace functionality to point it back to our defense unicorns fork?

**Comment by Racer159** [comment]
The README should be updated for sure - the copyrights will need to have xeipuuv attribution in them still, but could have `, and the gojsonschema authors` added - https://www.apache.org/licenses/LICENSE-2.0.html#redistribution

**Comment by Racer159** [comment]
And yes it would be to https://github.com/gojsonschema/gojsonschema so we can PR back to that repo

**Review by lucasrod16 — APPROVED** [review]
lgtm

File: schemaPool.go [hunk 2]
```diff
@@ -150,7 +150,7 @@ func (p *schemaPool) GetDocument(reference gojsonreference.JsonReference) (*sche
 	}
 
 	// Create a deep copy, so we can remove the fragment part later on without altering the original
-	refToURL, _ := gojsonreference.NewJsonReference(reference.String())
+	refToURL, _ := jsonreference.New(reference.String())
```
Comment (bdw617) [inline schemaPool.go:153]: please check the error here and it looks perfect.

**Review by bdw617 — CHANGES_REQUESTED** [review]

File: schemaPool.go (hunk 2 above)
Comment (bdw617) [reply schemaPool.go:153]: I'll let this go, gosec didn't complain

**Review by bdw617 — APPROVED** [review]

**Pull request merged**